platforms.go

```diff
@@ -174,9 +175,13 @@ func ParseAll(specifiers []string) ([]specs.Platform, error) {
 
 // Parse parses the platform specifier syntax into a platform declaration.
 //
-// Platform specifiers are in the format `<os>|<arch>|<os>/<arch>[/<variant>]`.
+// Platform specifiers are in the format `<os>[(<OSVersion>)]|<arch>|<os>[(<OSVersion>)]/<arch>[/<variant>]`.
 // The minimum required information for a platform specifier is the operating
-// system or architecture. If there is only a single string (no slashes), the
+// system or architecture. The OSVersion can be part of the OS like windows(10.0.17763)
+// Currently, the OS version is only used by windows. Therefore, if the OS is windows
```

Comment:
> if the OS is windows

There doesn't seem to be any windows-specific check (and I don't think there should be)